javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,48 @@
+//A future age calculator
+
+let yearOfBirth = 1982;
+let yearFuture = 2046;
+let age;
+age = yearFuture - yearOfBirth;
+console.log("\"You will be " + age + " years old in " + yearFuture + "\"");
+
+//A dog age calculator
+
+let dogYearOfBirth = 2000;
+let dogYearFuture = 2010;
+let dogYear = (dogYearFuture - dogYearOfBirth) * 4;
+let shouldShowResultInDogYears;
+if (shouldShowResultInDogYears = true) {
```

Comment:
Here you used the assignment operator `=` instead of the comparison operator `===`. This means `shouldShowResultInDogYears` will always equal to true and you won't ever be able to reach your else block. Very common mistake, just remember to keep an eye out for this next time.